chapter02/spg/question2.pl

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use v5.14;
+
+use Module::CoreList;
+use Business::ISBN;
+
+foreach my $module ( sort keys $Module::CoreList::version{5.014002} ) {
+  print $module, ' ' , Module::CoreList::first_release_by_date( $module ), "\n"
+}
```

Comment:
The question asks for the _"first release date"_ to be reported for `5.014002` modules, not the perl version they were first released in.